### DISCUSSION TITLE
Skin library

### DISCUSSION THREAD

**Discussion by andrey6498** [ppy/osu#17759] [Ideas] [answered]
Pretty much you can use same pattern as for distributing songs, and maybe even add another tab(or use same tab, adding 2nd section with skins).
 Reasons: 
- Recognition for skinners. 
- Add a place where you can find a skin to your liking easily.
- Possibly use download/like ratios for leaderboatds and etc.
- It can be nice environment for skin contests maybe?

I'm not sure what else to add here.

_Also theres a possibility that it was already suggested in the past. Im just blind in this case._

**Comment by peppy** [comment, marked as answer]
suggested many times, will happen one day
